pkgs/top-level/all-packages.nix

```diff
@@ -20971,7 +20971,7 @@ with pkgs;
   mod_timestamp = apacheHttpdPackages.mod_timestamp;
 
   inherit (callPackages ../servers/mpd {
-    stdenv = if stdenv.cc.isClang then llvmPackages_8.stdenv else stdenv;
+    stdenv = if (with stdenv; cc.isClang && isx86_64) then llvmPackages_8.stdenv else stdenv;
```

Comment:
This line can be dropped completely, since #148396 will upgrade clang to 11 on all darwin platforms.

Reply:
Sounds good, I'll force push to a more sensible changeset once #148396 gets merged (waiting so as not to trigger an ofborg eval where I know `x86_64-darwin` will fail).